app/javascript/util/directDebitDecider.js

```diff
@@ -1,15 +1,37 @@
 import { isObject, isMatch, find } from 'lodash';
 
 const SUPPORTED_COUNTRIES = [
-  { country: 'GB', recurring: true },
-  { country: 'NL', recurring: true },
-  { country: 'FR', recurring: true },
+  { country: 'GB' },
+  { country: 'NL' },
+  { country: 'FR' },
   { country: 'DE' },
   { country: 'AT' },
   { country: 'ES' },
   { country: 'AU' },
+  { country: 'BE' },
+  { country: 'CY' },
+  { country: 'EE' },
+  { country: 'IE' },
+  { country: 'IT' },
+  { country: 'LV' },
+  { country: 'LU' },
+  { country: 'MC' },
+  { country: 'PT' },
+  { country: 'SM' },
+  { country: 'SI' },
+  { country: 'CA' },
+  { country: 'US' },
```

Comment:
Did fundraising agree to this list? Does anything need to be done in the GoCardless dashboard to enable accounts for these countries?

Reply:
@TanyaPinto ⬆️

Reply:
@osahyoun 
I pick the countries list from GoCardless. So I think there would not be issue.
I am not adding the country list which faces frequent failures